frontends/p4/typeChecking/typeChecker.cpp

```diff
@@ -3646,31 +3714,30 @@ const IR::Node* TypeInference::postorder(IR::ConstructorCallExpression* expressi
         simpleType = type->to<IR::Type_SpecializedCanonical>()->substituted;
 
     if (simpleType->is<IR::Type_Extern>()) {
-        auto args = checkExternConstructor(expression, simpleType->to<IR::Type_Extern>(),
-                                           expression->arguments);
-        if (args == nullptr)
+        auto typeAndArgs = checkExternConstructor(expression, simpleType->to<IR::Type_Extern>(),
+                                                  expression->arguments);
+        auto contType = typeAndArgs.first;
+        auto newArgs = typeAndArgs.second;
+        if (newArgs == nullptr)
```

Comment:
```
auto [contType, newArgs] = checkExternConstructor(expression, simpleType->to<IR::Type_Extern>(),
                                                  expression->arguments);
```
(github did save my comment previously).

Reply:
The bazel build system complains about your proposal:
```
frontends/p4/typeChecking/typeChecker.cpp:1130:14: warning: decomposition declaration only available with -std=c++1z or -std=gnu++1z
         auto [type, newArgs] = checkExternConstructor(decl, et, decl->arguments);
              ^
```
Anyone know where the language level is in the bazel scripts?